examples/CMakeLists.txt

```diff
@@ -1,5 +1,16 @@
 SUBDIRS( HelloWorld BasicDemo )
 IF(BUILD_BULLET3)
+	SET(DL dl)
```

Comment:
CMake defines the [`CMAKE_DL_LIBS`](https://cmake.org/cmake/help/v3.0/variable/CMAKE_DL_LIBS.html) to avoid having OS-specific code in each project.